### DISCUSSION TITLE
How knows about S3 bucket?

### DISCUSSION THREAD

**Discussion by melody4482** [melodyxpot/speech-chatwithfiles#11] [Q&A] [answered]
Hello
I need to implement File storage feature using Node.js + s3 bucket.
But What is needed for that?
I am a beginner so please tell me

**Comment by nyx0521** [comment, marked as answer]
Hello there.
I am a super Node.js Expert so I can helpyou.
first, you should create s3 bucket in your AWS account.
And then, you can get the Bucket API key and secret key
There's a s3-sdk npm module.
You can use that feature.
Thanks.

**Comment by melody4482** [comment]
Thanks.
You are good man
